megamek/src/megamek/client/ui/displayWrappers/FontDisplay.java

```diff
@@ -49,12 +55,34 @@ public Font getFont() {
 
     public static List<FontDisplay> getSortedFontDisplays() {
         return Stream.of(GraphicsEnvironment.getLocalGraphicsEnvironment().getAvailableFontFamilyNames())
-                .map(familyName -> new FontDisplay(Font.decode(familyName).deriveFont(Font.PLAIN, 16f), familyName))
+                .map(FontDisplay::new)
                 .collect(Collectors.toList());
     }
 
     @Override
     public String toString() {
         return displayName;
     }
+
+    @Override
+    public boolean equals(final @Nullable Object other) {
+        if (other == null) {
+            return false;
+        } else if (this == other) {
+            return true;
+        } else if (other instanceof FontDisplay) {
+            return getFont().equals(((FontDisplay) other).getFont());
+        } else if (other instanceof Font) {
+            return getFont().equals(other);
```

Comment:
As someone who did not write this class I would never write fontDisplay.equals("Verdana") or fontDisplay.equals(someFont) and expect it to return true without taking a deep look at it first. Far be it from me to create unnecessary work but in this case I'd say isFont(String fontName) and isFont(Font font) would be easier on other users of this class such as me.

Reply:
Both were testing setups that don't compare properly (which lead to the constructor change), so I've removed them. I was trying to simplify selected item assignment by making them possible.